src/cim/components/Agent/tableUtils.tsx

```diff
@@ -168,6 +168,13 @@ export const infraEnvColumn = (
       if (infraEnvName && getInfraEnvLink) {
         title = getInfraEnvLink({ name: infraEnvName, namespace: agent.metadata?.namespace || '' });
       }
```

Comment:
`getInfraEnvLink` was meant to return the Link component but for some reason its not used at all. And I think we actually do not need it and can use `Link` directly as you did.

That means we want to
 - remove `getInfraEnvLink` and `getInfraEnvLinkType` from everywhere
 - change the code to  
```suggestion
      if (infraEnvName) {
        title = (
          <Link
            to={`/multicloud/infrastructure/environments/details/${agent.metadata?.namespace}/${infraEnvName}/overview`}
          >
            {`${title}`}
          </Link>
        );
```

Reply:
I've made the changes you suggested

Reply:
I've made the changes you suggested